hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -522,14 +523,18 @@ public static void main(String[] args) throws Exception {
      */
     private transient DeltaSync deltaSync;
 
+    private final HoodieDeltaStreamerConfig deltaStreamerConfig;
+
     public DeltaSyncService(Config cfg, JavaSparkContext jssc, FileSystem fs, Configuration conf,
                             Option<TypedProperties> properties) throws IOException {
+      this.props = properties.get();
       this.cfg = cfg;
       this.jssc = jssc;
       this.sparkSession = SparkSession.builder().config(jssc.getConf()).getOrCreate();
       this.asyncCompactService = Option.empty();
+      this.deltaStreamerConfig = new HoodieDeltaStreamerConfig(props);
 
-      if (fs.exists(new Path(cfg.targetBasePath))) {
+      if (fs.exists(new Path(cfg.targetBasePath)) && !deltaStreamerConfig.getFullOverwrite()) {
```

Comment:
is this configurable from the delta streamer command-line tool? Looks like we don't have this option there.

Reply:
1.'deltaStreamerConfig.getFullOverwrite()' is only for dfsSouce, is it inappropriate to put it in the delta streamer command-line tool?
2. There are more and more deltastreamer configurations, we can manage these in one class

Reply:
If a user would like to use overwrite through the command-line tool, is there any way to do that?

Reply:
The parameter itself only acts on DFSSouce